.github/workflows/lint.yml

```diff
@@ -0,0 +1,18 @@
+name: Lint
+
+on: [push]
+
+jobs:
+  lint:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.8
+
+    - name: Check precommit hooks
+      uses: pre-commit/action@v3.0.0
```

Comment:
Replace the "flake8" workflow with this one that will run the precommit hook on all files, and as such do flake8, black, isort and mypy